README.md

```diff
@@ -4,62 +4,73 @@ This is a collection of most of the freestyle tricks, for people to keep a bette
 
 The link to the page is: https://highline-freestyle.com
 
+
 ## Features
 
 - Predefinded tricks from the [google sheet](https://docs.google.com/spreadsheets/d/1amLK2b6BQkJ10I3LcbUe-D-wgQpHkcgoIrL10TPkHPo) **Feedback to tricks, on this sheet as a comment and not as an issue**
 - Tricks added by the user
 - Random combo generator
 - Possibility to mark tricks on how good you can do them
 
+
 ## Future plans
 
 - ComboMaker to allow the user to save his own combo
 - Possibility to share own tricks and combos with other people
 - Integrate judging feature for competitions
 
+
+
 # Get involved
 
 Help us out develop an awesome app!
 
+
 ## Bugs, suggestions, feedback
 
 Open an issue and describe what is you want to see different
 
+
 ## Development
 
 We don't always know what we really do, but we try to keep some order, through issues ...
 
 Checkout the [good first issue tag](https://github.com/bastislack/highline-freestyle/labels/good%20first%20issue), if you don't know where to start.
 
+
 ### Requirements:
 
-- node
-- npm
+- Node (v20.6.1)
+- npm (v9.8.1)
```

Comment:
Any reason this is not the latest version, `v10.1.0`?

Reply:
I chose it because it is the one that comes with Node v20.6.1, so if someone installs Node version 20.6.1 the person should have npm version 9.8.1 installed by default and they will work well together. I got the info from [this StackOverflow answer](https://stackoverflow.com/a/51243675) referencing [this json file](https://nodejs.org/dist/index.json) (check first item of the file). The info also checks out on my machine.

Reply:
Ye. Personally I don't see the benefit in providing a specific npm version. We dont make use of any recent npm version's features — so as long as you did update your npm version in the last 5 yrs or so you should be good :D